code/webauthn/contracts/deploy/deploy.ts

```diff
@@ -0,0 +1,64 @@
+import { utils, Wallet, Provider } from 'zksync-ethers';
+import type { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { Deployer } from '@matterlabs/hardhat-zksync';
+// load env file
+import dotenv from 'dotenv';
+import { ethers } from 'ethers';
+dotenv.config();
+
+const DEPLOYER_PRIVATE_KEY = process.env.WALLET_PRIVATE_KEY || '';
+
+export default async function (hre: HardhatRuntimeEnvironment) {
+  // @ts-expect-error target config file which can be testnet or local
+  const provider = new Provider(hre.network.config.url);
+  const wallet = new Wallet(DEPLOYER_PRIVATE_KEY, provider);
+  const deployer = new Deployer(hre, wallet);
```

Comment:
with the current version there's no need to initialise a deployer instance. You can use `hre.deployer` instead

See https://docs.zksync.io/build/tooling/hardhat/plugins/hardhat-zksync-deploy#transition-from-deployer-object-to-the-hredeployer